### ISSUE TITLE
Select some features and display them

### ISSUE THREAD

**Issue by txomon** [pditommaso/awesome-pipeline#80]
Hello, when looking at all these projects, I am seeing that they are really different in nature. There seem to be just cluster execution frameworks, others are more DAG based orchestrators, and there are some that are just single threaded local executors.

I can't say exactly the name I would use, but I would definitely see interesting to know what kind of features they have, like backends they rely on, if they are DSL based or an specific language, etc.

**Comment by pditommaso** [comment]
Yes, I've realised that. This repository grew much beyond my expectations. I support a re-organisation as you are suggesting, but I definitely don't have the time to look at that.
